containers/Socios/UsersContainers.tsx

```diff
@@ -11,7 +11,7 @@ interface PaginatedProps {
 
 const UsersContainers: FC<PaginatedProps> = ({ users }) => {
   const [CurrentPage, setCurrentPage] = useState(1)
-  const PartnersPerPage = 12
+  const PartnersPerPage = 5
```

Comment:
Why change this? I think 12 is fine

Reply:
Because there are problems with css of the pagination that another developer added that I need to make bigger changes to fix it and he is still working with another issue right now related to what I'm doing as well. So maybe when he finish the pagination. We could try to make another issue to fix the way that the pagination works. 
https://github.com/Azordev/did-admin-panel/pull/166

If I put 12 then it gives this result:

https://user-images.githubusercontent.com/77207702/188722807-4fdf7e61-b401-46fa-bd7e-809d7d9ddd20.mov

Reply:
My only question is if the pagination will be only be part of tables or it will also work with lists like in the events view.

Reply:
@Eghinner @e-azocar @angelik0828 @elias-miguel-vergara please check, seems like we need to create a new bug

For the time being, we can merge with `PartnersPerPage = 5` but we have to change to 12 after the fix

Reply:
yes, another issue